src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -139,12 +140,13 @@ public void RunAndBlock(CancellationToken cancellationToken = default(Cancellati
                     lock (_liveInstances)
                     {
                         _liveInstances.Add(newInstance);
+                        _hostRestartCount++;
                     }
 
                     OnHostCreated();
 
-                    string message = string.Format("Starting Host (HostId={0}, Version={1}, ProcessId={2}, Debug={3}, Attempt={4}, FunctionsExtensionVersion={5})",
-                        newInstance.ScriptConfig.HostConfig.HostId, ScriptHost.Version, Process.GetCurrentProcess().Id, newInstance.InDebugMode.ToString(), consecutiveErrorCount,
+                    string message = string.Format("Starting Host (HostId={0}, Version={1}, ProcessId={2}, Debug={3}, ConsecutiveErrors={4}, TotalRestarts={5}, FunctionsExtensionVersion={6})",
```

Comment:
nit: The number of parameters here is starting to get to the point where string interpolation would make this a lot more readable

Reply:
Added string interpolation